src/ILLink.RoslynAnalyzer/Resources.resx

```diff
@@ -117,10 +117,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="RequiresUnreferencedCodeAnalyzerTitle" xml:space="preserve">
-    <value>RequiresUnreferencedCodeAnalyzer</value>
+  <data name="RequiresUnreferencedCodeTitle" xml:space="preserve">
+    <value>Avoid calling members marked with `RequiresUnreferencedCodeAttribute` when trimming application code</value>
```

Comment:
```suggestion
    <value>Avoid calling methods marked with 'RequiresUnreferencedCodeAttribute' when trimming application code</value>
```

Also I'm not sure that something marked with this attribute is something that the user should avoid calling, it's just a method that requires dynamic access, which is dangerous when linking the application (but I really don't have a good suggestion for framing this title).

Reply:
We could rephrase by saying something like "Members annotated with `RequiresUnreferencedCodeAttribute` may behave differently or fail if the unreferenced code is removed by trimming"

I think that's a decent description of the problem, but it doesn't really give a call to action, like "don't call this when trimming." I don't have a strong opinion either way, but I think we should decide on what kind of title we want to present.

The context for this resource is that when you're looking at a list of the warnings, this will be the description of what the warning is about. The message is what's actually printed when the warning fires.

Reply:
In a previous commit, I changed the message to:
Methods marked with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code